module/VuFind/src/VuFind/Auth/Shibboleth.php

```diff
@@ -105,23 +174,30 @@ protected function validateConfig()
      */
     public function authenticate($request)
     {
+        // validate config before authentication
+        $this->validateConfig();
```

Comment:
The `validateConfig` method should be called automatically whenever the built-in `getConfig` method is called (based on logic set up in the authentication base class). Does the introduction of the configuration loader require us to revisit this in any way? I just want to be sure that `validateConfig` is validating the right thing here, and also checking whether there's a more natural place where we can ensure that validation is occurring automatically whenever it is needed. Let me know if you need me to do any deeper analysis on these questions; it's possible that there is no problem here, but this change led me to raise the questions just to be sure we're doing things correctly.